rosidl_typesupport_microxrcedds_c/resource/msg__type_support_c.c.em

```diff
@@ -317,6 +344,17 @@ size_t get_serialized_size_@('__'.join([package_name] + list(interface_path.pare
     size_t item_size = sizeof(ros_message->@(member.name).data[0]);
     current_alignment += ucdr_alignment(current_alignment, MICROXRCEDDS_PADDING) + MICROXRCEDDS_PADDING;
     current_alignment += ucdr_alignment(current_alignment, item_size) + (sequence_size * item_size);
+@[      elif isinstance(member.type.value_type, NamespacedType)]@
+    const size_t sequence_size = ros_message->@(member.name).size;
+
+    current_alignment += ucdr_alignment(current_alignment, MICROXRCEDDS_PADDING) + MICROXRCEDDS_PADDING;
+
+    size_t item_size = ((const message_type_support_callbacks_t *)(
```

Comment:
Could there be items with different size?

Reply:
Please check the new approach

Reply:
What happens when `get_serialized_size(&ros_message->@(member.name).data[0]) != get_serialized_size(&ros_message->@(member.name).data[1])`

Reply:
It iterates along the sequence adding each element size:

https://github.com/micro-ROS/rosidl_typesupport_microxrcedds/pull/10/files#diff-12f1da91c0bce55270842a34a0d90994R359

I wonder if [this](https://github.com/micro-ROS/rosidl_typesupport_microxrcedds/pull/10/files#diff-12f1da91c0bce55270842a34a0d90994R363) alignment is neccesary.

Reply:
Sorry, but I show an outdated diff.